src/playlist.js

```diff
@@ -236,6 +236,21 @@
                             includeTrailingTime);
   };
 
+  liveTargetDurations = function(playlist) {
+    if (!playlist.liveTargetDurations_) {
+      playlist.liveTargetDurations_ = 3;
+    }
+    return playlist.liveTargetDurations_;
+  };
+
+  setLiveTargetDurations = function(playlist, numTargetDurations) {
+    if (numTargetDurations > 1) {
+      playlist.liveTargetDurations_ = numTargetDurations;
+    } else {
```

Comment:
@gkatsev To answer your question, this was basically the reason for the getter/setter structure. I want to make sure this value is at least 1, since making it anything less doesn't make sense and breaks the code. It seemed easier/smarter to police the value in a setter than in every place it's referenced forever and always.